Interop/ModelCard/src/main/java/ModelCard.java

```diff
@@ -0,0 +1,74 @@
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import org.tribuo.Model;
+import org.tribuo.evaluation.Evaluation;
+import org.tribuo.interop.ExternalModel;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Paths;
+
+public class ModelCard {
+    private static final ObjectMapper mapper = new ObjectMapper().enable(SerializationFeature.INDENT_OUTPUT);
```

Comment:
Maybe this could be package private and then all the classes can share the same instance?

Reply:
Done as suggested.